otter/cloud_client.py

```diff
@@ -1051,3 +1051,62 @@ def publish_to_cloudfeeds(event, log=None):
             'content-type': ['application/vnd.rackspace.atom+json']},
         data=event, log=log, success_pred=has_code(201),
         json_response=False)
+
+
+# ----- Cloud orchestration requests -----
+def list_stacks_all(parameters=None):
+    """
+    List Heat stacks.
+
+    :ivar dict parameters: Query parameters to include.
+
+    Succeed on 200.
+
+    :return: a `tuple` of (:obj:`twisted.web.client.Response`, JSON `dict`)
+    """
+
+    # TODO Actually implement batch handling
```

Comment:
I don't think there's any particular performance concern, I'm more worried about correctness. If the Heat API batches by default, and we're only processing the first batch, then otter will think that those stacks don't exist when they actually do.

I've looked at both the OpenStack Heat API docs (http://developer.openstack.org/api-ref-orchestration-v1.html) and the Rackspace ones (https://developer.rackspace.com/docs/cloud-orchestration/v1/developer-guide/) and neither of them document the pagination behavior of a query without `limit` supplied, so I don't know what will happen.